### ISSUE TITLE
Call gpg_startup only after setup_mosrs_account succeeds

### ISSUE THREAD

**Issue by penguian** [penguian/mosrs-setup#46]
The call to `setup.gpg_startup` should be made only after the user's MOSRS account credentials have been set up successfully. At this point, the GPG environment variables are known to exist, so the functions `setup.check_gpg_startup` and `gpg.get_environ` are no longer needed.
